### PR TITLE
MXLoginSSOFlow: Use unstable identity providers field while the MSC2858 is not approved

### PR DESCRIPTION
**Pull request by SBiOSoftWhare**
See https://github.com/matrix-org/matrix-doc/pull/2858/#discussion_r543567196 for more details.

### CODE REVIEW

**Comment by bmarty** [comment]
LGTM. The URL change will be done in Element iOS I assume?

**Review by bmarty — APPROVED** [review]

**Comment by SBiOSoftWhare** [comment]
> LGTM. The URL change will be done in Element iOS I assume?

Yes

File: MatrixSDK/JSONModels/Login/MXLoginSSOFlow.m [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 #import "MXLoginSSOFlow.h"
 
-NSString *const MXLoginSSOFlowIdentityProvidersKey = @"identity_providers";
+NSString *const MXLoginSSOFlowIdentityProvidersKey = @"org.matrix.msc2858.identity_providers";
```
Comment (bmarty) [inline MatrixSDK/JSONModels/Login/MXLoginSSOFlow.m:19]: Maybe create another key: `MXLoginSSOFlowMsc2858IdentityProvidersKey`. In the future `MXLoginSSOFlowIdentityProvidersKey` will have to coexist with the value `"identity_providers"`.
But this change could be done later

**Comment by bmarty** [comment]
Perfect, thanks

**Pull request merged**